### ISSUE TITLE
Let plugin handle Buy Back Pledges page too

### ISSUE THREAD

**Issue by Petosiris** [dolkensp/HangarXPLOR#25]
It'd be nice if HangarXPLOR could work on the Buy Back Pledges page too, for some functions at least. I'm thinking of - more than 100 per page, filter, sort, and compact listing, to begin with.

**Comment by CJ102168** [comment]
This feature would be welcomed. Would love to have a search bar with it by default.

**Comment by cintaracorp** [comment]
Yes, just gathering all the buyback pledges on a single page would be a huge benefit. Right now we need to step through them, 10 pledges at a time, making it a real pain! Even a 100 per page setting would help a lot!

**Comment by Petosiris** [comment]
I agree, of course, since it was I who suggested this feature, but after three years I'm not exactly holding my breath while waiting for it to be implemented.  I think this project needs more code contributers.
